static/js/components/subject_page/block.tsx

```diff
@@ -816,21 +833,24 @@ function renderWebComponents(
               : {})}
           />
         );
-      case "SCATTER":
+      case "SCATTER": {
+        const statVarSpec = props.statVarProvider.getSpecList(tile.statVarKey, {
+          blockDate,
+          blockDenom,
+        });
+        const title = blockDenom
+          ? addPerCapitaToVersusTitle(tile.title, statVarSpec)
+          : tile.title;
         return (
           <datacommons-scatter
             key={id}
             id={id}
-            header={tile.title}
+            header={title}
```

Comment:
I might be missing it, but should this have a `addPerCapitaToVersusTitle`?

Reply:
Is line 841-843 sufficient, or is there a better approach?

Reply:
Oh whoops I missed that 😂